components/molecules/ContributionsTreemap/contributions-treemap.stories.tsx

```diff
@@ -5,7 +5,7 @@ import { ContributionsTreemap } from "./contributions-treemap";
 type Story = StoryObj<typeof ContributionsTreemap>;
 
 const meta: Meta<typeof ContributionsTreemap> = {
-  title: "Components/Graphs/ContributionsTreemap",
+  title: "components/molecules/ContributionsTreemap",
```

Comment:
This isn't really in the scope of this PR but since it was blocking the ability to pass the CI checks, I updated the title here to resolve the duplicate id issue mentioned in this comment here 

https://github.com/open-sauced/app/pull/2535#issuecomment-1911329089